src/llmq/ehf_signals.cpp

```diff
@@ -133,7 +137,7 @@ void CEHFSignalsHandler::HandleNewRecoveredSig(const CRecoveredSig& recoveredSig
         LOCK(cs_main);
         const MempoolAcceptResult result = AcceptToMemoryPool(chainstate, mempool, tx_to_sent, /* bypass_limits */ false);
         if (result.m_result_type == MempoolAcceptResult::ResultType::VALID) {
-            connman.RelayTransaction(*tx_to_sent);
+            connman.RelayTransaction(*tx_to_sent, static_cast<bool>(m_cj_ctx->dstxman->GetDSTX(tx_to_sent->GetHash())));
```

Comment:
`ehf_signals` should not know anything about DSTX. Its transactions are never CJ related.

Reply:
Resolved in latest push.